cmd/agent/app/builder_test.go

```diff
@@ -125,7 +125,7 @@ func TestBuilderWithProcessorErrors(t *testing.T) {
 	}{
```

Comment:
Please add a test case to TestBuilderFromConfig where the new parameter is not 0. This should take care of the low code coverage for the diff.